### PR TITLE
css sorting and intro improvements

### PR DESCRIPTION
**Pull request by samioan**
-Each component has its own css file, for individual styling
-Added portfolio buttons on intro page
-Changed favicon and index.html file
-Deleted unnecessary faces images

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
     "lodash": "^4.17.20",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-native": "^0.63.4",
+    "react-native-web": "^0.14.10",
```
Comment (stacon) [inline package.json]: React Native is for Android IOS app development. What's it's usage here?

**Review by stacon — CHANGES_REQUESTED** [review]
Probably react-native and react-native-web were added by mistake. Remove these entries. Do a `npm install` and make sure that the app works afterwards.

File: package.json (hunk 1 above)
Comment (samioan) [reply package.json]: For the Linking and BackHandler functions used in the intro component.
If they are not needed and can be done in another way, I'll remove them.

Comment (stacon) [reply package.json]: If its about handling the back button there should probably be away through react-router,

**Review by stacon — APPROVED** [review]

**Pull request merged**